.ci/build_kubernetes.sh

```diff
@@ -59,9 +60,10 @@ docker image pull ${DOCKER_USERNAME}/elasticsearch:${DOCKER_ELASTIC_IMAGE_TAG} |
 )
 
 # Replace docker images for php-fpm of application
-yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.containers[0].image ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG}
 yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.initContainers[0].image ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG}
 yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.initContainers[1].image ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG}
+yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.containers[0].image ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG}
+yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.containers[2].image ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG}
```

Comment:
When I saw this, the first question on my mind was *Why are you changing the image of the first and third container? What about the second one?*

Then I've realized that the second container is *nginx*... I'd definitely love to see #1271 resolved before we do any more changes to the Kubernetes manifests. This is a mistake waiting to happen...

Reply:
My thoughts exactly